x/airdrop/keeper/airdrop_account.go

```diff
@@ -55,6 +57,27 @@ func (k Keeper) GetAllAirdropAccounts(
 	return
 }
 
+func (k Keeper) CreateAirdropAccount(
+	ctx sdk.Context,
+	airdropAccount *types.AirdropAccount,
+) (err error) {
+	if airdropAccount.OriginAccountCreated {
+		return fmt.Errorf("origin account already created")
+	}
+	if err = k.CreateOriginAccount(ctx, airdropAccount); err != nil {
+		return
+	}
+	if err = k.MintOriginTokens(ctx, airdropAccount); err != nil {
+		return
+	}
+	if err = k.SendOriginTokens(ctx, airdropAccount); err != nil {
+		return
+	}
+	airdropAccount.OriginAccountCreated = true
+	err = k.SetAirdropAccount(ctx, airdropAccount)
+	return
```

Comment:
just a styling nitpic
```suggestion
		return err
	}
	if err = k.MintOriginTokens(ctx, airdropAccount); err != nil {
		return err
	}
	if err = k.SendOriginTokens(ctx, airdropAccount); err != nil {
		return err
	}
	airdropAccount.OriginAccountCreated = true
	return k.SetAirdropAccount(ctx, airdropAccount)
```

Reply:
I'm not totally sold that this is better but that last line is a good update. I'll update all of these since I don't have a hard preference either way 👍🏻

Reply:
https://github.com/ojo-network/ojo/pull/175/commits/c6c627eb3b4f97d06f59afe4e7bced6090e8d528